docs/source/asr/speaker_diarization/configs.rst

```diff
@@ -18,12 +15,12 @@ Example configuration files for speaker diarization can be found in the
 
 
 Dataset Configuration
```

Comment:
Outdated. Please update

Reply:
I think you referred to the old version. It is now `<NeMo_git_root>/examples/speaker_tasks/diarization/conf/offline_diarization.yaml`

Reply:
At that time I thought the whole dataset config was outdated. I will go over it again if you've updated it.